packages/webdriverio/src/constants.js

```diff
@@ -422,3 +421,9 @@ export const APPIUM_CAPABILITES = [
     ...APPIUM_ANDROID_CAPABILITIES,
     ...APPIUM_IOS_CAPABILITIES
 ]
+export const DRIVER_DEFAULT_ENDPOINT = {
+    method: 'GET',
+    host: 'localhost',
+    port: 4444,
+    path: '/status'
```

Comment:
path should respect the `path` defined by user in config! For selenium v3 and some cloud providers it should be /wd/hub/status

I've just verified it locally

Reply:
by default all drivers should start with path `/` if you change that we should assume you run `webdriver`. I modified the `getAutomationProtocol` method to now take this into consideration:

```js
/**
 * run WebDriver if hostname or port is set
 */
if (config.hostname || config.port || config.path || (config.user && config.key)) {
    return 'webdriver'
}
```